frontend/components/Card.tsx

```diff
@@ -12,7 +13,25 @@ const StyledCard = styled(AntdCard)`
   flex-direction: column;
   font-size: 0.8rem;
   padding: 1rem;
-  margin: 0.25rem;
+  width: 30%;
+  max-width: 281px;
+  margin-right: min(3%, 39px);
+
+  @media ${maxDevice.laptop} {
+    flex-grow: 1;
+    width: 45%;
+    max-width: 100%;
+    margin: 0.25rem;
+    height: 138px;
```

Comment:
make it related to `vh` for responsiveness

Reply:
actually, lets get rid of the line `height: 138px;` 
that'll make the cards more responsive (not tied to a pixel value)